doc/examples/cloud-config-yum-repo.txt

```diff
@@ -3,16 +3,18 @@
 #
 # Add yum repository configuration to the system
 #
-# The following example adds the file /etc/yum.repos.d/epel_testing.repo
+# The following example adds the file /etc/yum.repos.d/epel.repo
 # which can then subsequently be used by yum for later operations.
+# Either baseurl or metalink can be used but not both
```

Comment:
This is wrong. Both can be used also, see https://linux.die.net/man/5/yum.conf 
```
 This can be used instead of or with the baseurl option.
```
What is incorrect is not to use either. At least one of the two must be present.